review-tools/ghmerge

```diff
@@ -130,7 +130,7 @@ else
 fi
 ADDREVOPTS=${ADDREVOPTS# } # chop any leading ' '
 
-[ "$REMOTE" = "" ] && REMOTE=`git remote -v | awk '/git.openssl.org.*(push)/{ print $1; }' | head -n 1` # usually this will be 'upstream'
+[ "$REMOTE" = "" ] && REMOTE=`git remote -v | awk '/github.openssl.org:openssl/openssl.*(push)/{ print $1; }' | head -n 1` # usually this will be 'upstream'
```

Comment:
Please do not add openssl/openssl as the ghmerge was usable with web and tools repos as well before. Just checking for github.openssl.org should suffice.

Reply:
Or rather `github.openssl.org:openssl` to exclude personal forks.

Reply:
> Or rather `github.openssl.org:openssl` to exclude personal forks.

Nope, that won't work as tools are in otc/tools in GHE.

Reply:
How about `github.openssl.org:(openssl|otc)`? Don‘t know whether personal forks will be used on GHE, but it will be more robust.

Reply:
(Probably backslashes are required before the regex operators, didn‘t check)

Reply:
`github.openssl.org:(openssl|o[mt]c)` would be better.

general-policies is omc not otc.

Reply:
Sorry for forgetting about the OMC repos! But I would prefer `github.openssl.org:(openssl|omc|otc)` for better readability. Don't overengineer the regex ;-).

Reply:
I left the leading `o` in place at least :)

Reply:
You can bet that I also already thought about this further optimization. ;-)